src/bin/main.rs

```diff
@@ -0,0 +1,44 @@
+pub(crate) mod commands;
+pub(crate) mod config;
+use clap::{App, Arg};
+
+#[tokio::main]
+async fn main() {
+    let matches = App::new("Rusk")
+        .version("v0.1.0")
```

Comment:
As discussed, here you could have something like:
```rust
.version(show_version().as_str())
```

Where `show_version` is:
```rust
#[inline]
pub fn show_version() -> String {
    let info = rustc_tools_util::get_version_info!();

    let version = format!("{}.{}.{}", info.major, info.minor, info.patch);
    let build = format!(
        "{} {}",
        info.commit_hash.unwrap_or_default(),
        info.commit_date.unwrap_or_default()
    );

    let version_build = if build.len() > 1 {
        format!("{} ({})", version, build)
    } else {
        version
    };

    version_build
}
```

Or something like that.
This is needed because unfortunately both `Clap` and `rustc_tools_utils` add the crate's name, so you'll end up to have it duplicated.

Since you're not doing a cargo subcrate command, one thing you could do is actually get the `info` object in the `main`, and pass it to `show_version` instead, so you can also use `App::new(&info.crate_name)` or similar if you want automatically get the crate's name.

Reply:
Good idea! Applied all of the changes!